src/http/api/statistics.js

```diff
@@ -9,13 +9,17 @@ module.exports = class Statistics extends Route {
     }, client)
   }
 
-  register (app) {
+  async register (app) {
     const router = Router()
+    const shardGuildCounts = await this.client.shard.fetchClientValues('guilds.cache.size')
+    const totalGuildCount = shardGuildCounts.reduce((total, current) => total + current)
+    const shardUserCounts = await this.client.shard.fetchClientValues('users.cache.size')
+    const totalUserCount = shardUserCounts.reduce((total, current) => total + current)
```

Comment:
```suggestion
    const shardUserCounts = await this.client.shard.broadcastEval('this.guilds.cache.reduce((acc, g) => acc + g.memberCount, 0)')
    const totalUserCount = shardUserCounts.reduce((total, current) => total + current, 0)
```

Reply:
Using `#Guild.memberCount` in this situation isn't bad? Because you're counting the same user twice or more if they are in more than one guild and etc.

Reply:
Not sure, because the memberCount is a accurate method to get how many members are in a server, because even when I fetch all members `memberCount` =/= `members.cache.size`

Reply:
This is the best and fastest way of getting the total amount of members the bot has access to

> Because you're counting the same user twice or more if they are in more than one guild and etc.

I don't see any issues with this